src/System.Private.CoreLib/shared/Interop/Windows/Interop.Errors.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal partial class Errors
+    {
+        internal const int ERROR_SUCCESS = 0x0;
+        internal const int ERROR_INVALID_FUNCTION = 0x1;
+        internal const int ERROR_FILE_NOT_FOUND = 0x2;
```

Comment:
The CoreCLR Interop.Errors.cs has a lot of unused ones. The CoreRT one is much closer to what is actually used.